apps/XMRig/install-32

```diff
@@ -0,0 +1,200 @@
+{
+    "autosave": true,
+    "cpu": true,
+    "opencl": false,
+    "cuda": false,
+    "pools": [
+        {
+            "url": "${pool_pref}",
+            "user": "${wallet_address}",
+            "rig-id": "${miner_name}",
+            "keepalive": ${keepalived},
+            "tls": ${tls}
+        }
+    ]
+}
+EOT
+}
+
+# Hardware and kernel checks
+
+if grep -q ARMv6 /proc/cpuinfo; then
+  error "XMRig is not compatible with the Raspberry Pi Zero or other ARMv6 boards."
+fi
+
+if [ "$(uname -m)" != "aarch64" ] && [ "$(uname -m)" != "arm64" ]; then
+    while [ "$check" == "0" ]; do
+      read -n 3 -p 'You need a 64-bit kernel to install XMRig. Do you want me to enable 64-bit kernel? [Y/n] ' input
+      case $input in
+              Y|y|yes|Yes|YES) check=1; enable_64bit ;;
+              N|n|no|No|NO) check=1; error "User error: 64-bit kernel is required but user chose to not enable it." ;;
+              *) check=0; echo 'Invalid input! Please answer y/n, or yes/no.' ;;
+      esac
+    done
+elif [ "$(uname -m)" == "arm64" ] || [ "$(uname -m)" == "aarch64" ]; then
+    status 'You have a 64-bit kernel. Continuing...'
+fi
+# Setup chroot
+
+install_packages debootstrap
+mkdir -p $HOME/xmrig/
+sudo debootstrap --variant=buildd --arch arm64 bullseye $HOME/xmrig/bullseye http://ftp.us.debian.org/debian/ || error 'Failed to debootstrap chroot environment!!!'
+chroot_dir="$HOME/xmrig/bullseye"
+sudo mkdir -p $chroot_dir/home/$USER/ $chroot_dir/root/.config
+sudo cp /etc/resolv.conf $HOME/xmrig/bullseye/etc/resolv.conf || error "Failed to copy /etc/resolv.conf' into chroot environment!"
+echo "export LANG=C" &>/dev/null | sudo tee --append $HOME/xmrig/bullseye/root/.bashrc || error "Failed to echo 'export LANG=C' to chroot environment's .bashrc"
+
+status 'Generating chroot XMRig install script...'
+sudo touch $chroot_dir/chroot-xmrig-inst.sh
+sudo cat << EOT | sudo tee $chroot_dir/chroot-xmrig-inst.sh || error 'Failed to create XMRig install script!'
+#!/bin/bash
+version=$version
+apt-get update
+#install xmrig
+error() { #red text and exit 1
+  echo -e "\e[91m$1\e[0m" 1>&2
+  exit 1
+}
+
+apt install git build-essential cmake make libuv1-dev libssl-dev libhwloc-dev -y || error 'Failed to install dependecies!'
+git clone --branch v$version https://github.com/xmrig/xmrig || error 'Failed to clone repository!'
+mkdir /xmrig/build && cd /xmrig/build || error 'Failed to create and cd into build directory!'
+cmake .. || error 'Failed to generate build files!'
+make -j$(nproc) || error 'Failed to build XMRig!'
+cp xmrig /usr/bin/xmrig
+EOT
+sudo chmod 775 $chroot_dir/chroot-xmrig-inst.sh
+status_green 'Done.'
+# Run install script
+sudo chroot $HOME/xmrig/bullseye ./chroot-xmrig-inst.sh
+
+# Setup xmrig command
+until [ "$check" == "1" ]; do
+read -p "Do you want to setup your xmrig.json? This will make running xmrig easier. [Y/n] " runwiz_pref
+read -p "I will now run the config helper. Is this ok? [Y/n] " runwiz_conf
```

Comment:
If the user answers "No" to the first question, they will still be presented with the second question.

Reply:
Really? In the case that the user says no, it should just continue with the rest of the script..

Reply:
@Botspot Sorry, I see what you're saying now.. if you say no, it'll just continue.. Fixed in an upcoming commit..

Reply:
Should be fixed with the latest commit 7227ccd.